README.md

```diff
@@ -17,13 +17,13 @@ npm i @bayer/ol-kit ol@4.6.5 react react-dom styled-components --save
 ## Getting Started
 It's easy to start building map apps with ol-kit. For simple projects the following will get you started:
 ```javascript
-import React from 'react'
+import { Component } from 'react'
```

Comment:
@glenselle this is gonna throw an error, `React` also has to be imported off the root of `react` before you can use `Component` -> `import React, { Component } from 'react'`